core-build/gulp-core-build/src/tasks/JestReporter.ts

```diff
@@ -0,0 +1,20 @@
+/* tslint:disable-next-line:no-any */
+const DEFAULT_REPORTER: any = require('jest-cli/build/reporters/DefaultReporter');
+
+/**
+ * Jest logs message to stderr. This class is to override that behavior so that
+ * rush does not get confused.
+ */
+/* tslint:disable-next-line:no-any */
+class JestReporter extends (DEFAULT_REPORTER as { new (globalConfig: any): any }) {
```

Comment:
> (DEFAULT_REPORTER as { new (globalConfig: any): any }) [](start = 26, length = 55)

What is this?  Is there some way we can make this into normal TypeScript code?

Reply:
This is a hack to override the default Jest behavior that writes everything to stderr.

---
In reply to: [145562766](https://github.com/Microsoft/web-build-tools/pull/391#discussion_r145562766) [](ancestors = 145562766)